DispatcherService/db/mongoose.js

```diff
@@ -1,7 +1,11 @@
-var mongoose = require('mongoose');
+require('dotenv').config()
+const mongoose = require('mongoose');
+const logger = require('./../logger/logger').logger;
 
 mongoose.Promise = global.Promise;
```

Comment:
mongoose has built in promises, you don't have to do this if you are using ES6 Promises which is the default that comes with node v8